.editorconfig

```diff
@@ -0,0 +1,27 @@
+# editorconfig.org
+
+# This is the project's root directory
+root = true
+
+[{*, .*}]
+# Use spaces for indentation
+indent_style = space
+# Each indent should contain 2 spaces
+indent_size = 2
+# Use Unix line endings
+end_of_line = lf
+# The files are utf-8 encoded
+charset = utf-8
+# No whitespace at the end of line
+trim_trailing_whitespace = true
+# A file must end with an empty line - this is good for version control systems
+insert_final_newline = true
+# A line should not have more than this amount of chars (not supported by all plugins)
+max_line_length = 100
```

Comment:
You can safely remove this one, because line length is handled by `Prettier` formatting.